tests/Unit/ImageTest.php

```diff
@@ -75,4 +79,216 @@ public function ensureUnprocessedFilesAreNotAvailable(Media $media)
 
         $getDerivativeResponse->assertNotFound();
     }
+
+    /**
+     * Provides Transformation string scenarios. contextually used in web requests for retrieving derivatives.
+     *
+     * @return array
+     */
+    public static function provideTransformationStrings(): array
+    {
+        return [
+            'valid_Quality' => [
+                'input' => 'q-50',
+                'expectedException' => null,
+                'expectedArray' => [
+                    'q' => 50,
+                ]
```

Comment:
```suggestion
                ],
```

Reply:
This is not done automatically by formatting, so I won't apply it.